docs/Installation.md

```diff
@@ -52,6 +52,12 @@ for Zonemaster::Backend, see the [declaration of prerequisites].
 > **Note:** In addition to the normal dependencies, the post-installation
 > smoke test instruction assumes that you have curl installed.

```

Comment:
These two lines are not needed since that is repeated for each OS.

Reply:
The scope of this PR is to update the smoke test. It's not about fixing all problems we can find in the installation instruction.